lib/v2/hrbust/maintainer.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    '/jwzx': ['LenaNouzen']
```

Comment:
```suggestion
    '/jwzx/:type?/:page?': ['LenaNouzen']
```
This should be the same as `router.js`